### DISCUSSION TITLE
Dziś jest (riposta 2)

### DISCUSSION THREAD

**Discussion by zaba-monika** [MajkiIT/polish-ads-filter#17774] [General] [answered]
A więc tak, podsumujmy krótko:
- dwóch pajaców założyło sobie wątek https://github.com/MajkiIT/polish-ads-filter/discussions/17449 w którym mnie trolują i wypisują na mnie jakies swoje wyssane z palca farmazony i mnie obrażają
- więc założyłam swój taki sam wątek https://github.com/MajkiIT/polish-ads-filter/discussions/17640 w którym odpisuję im na te ich farmazony, oczywiście robię to na spokojnie, bez wyzwisk, rozkładam ich na argumenty...
 - jak widać po mojej ostaniej ripoście, w której definitywnie udowodniłam, że są trollami, które się do mnie przyczepiły i nie chcą odczepić...tak ich zabolała prawda i tak im zabrakło argumentów, że xxcriticxx zablokował mój wątek bez powodu:
![sssss](https://user-images.githubusercontent.com/72980128/99904051-d85d3300-2cc8-11eb-84cd-fdd097f104a6.png)
- czyli: ich wątek w którym mnie obrażają i trolują nie został zablokowany i wszystko jest ok, ale mój wątek w którym jedynie się broniłam został chamsko zablokowany, to się nazywa moderatorska sprawiedliwość, a raczej moderatorska chamówka...
- oczywiście po zablokowaniu mi wątku (lub na szybko przed jego zablokowaniem) , kiedy nie mogę już zripostować, kopią mnie leżącą, niemogącą się bronić i odpisać, rzucając we mnie kolejnym mięsem pełnym manipulacji i kłamstwa: https://github.com/MajkiIT/polish-ads-filter/discussions/17449#discussioncomment-135678 :

> hawkeye116477 :
> @zaba-monika Nic nie napisałem, że chodzi o ciebie, ale skoro uważasz, że ciebie się tyczyło, to znaczy, że zaczynasz dostrzegać w sobie jakiś problem, z którym nie potrafisz sobie poradzić. Cóż, miejmy nadzieję, że kiedyś odnajdziesz kogoś, kto będzie potrafił ci pomóc i zmienisz się na lepsze. A na razie proponuję zrobić przerwę od GitHuba i zamiast tego może pograć w Wieśka 3 albo jakąś platformówkę dla relaksu, np GRIS.

- jakby nie było wiadomo o kogo chodzi w waszych komentarzach...ciągle mnie zaczepiacie i trolujecie...a potem głupio się wykręcasz i manipulujesz, że niby nie mnie chodziło..i jeszcze mi ubliżasz swoimi durnymi tekstami na dokładkę do swoich manipulacji...naprawdę wierzysz w to co sam piszesz? 
- i co znowu twój kolega xxcriticxx znowu mi zablokuje kolejną dyskusję bo znowu was prawda zabolała i na argumenty przegraliście tak jak w moim poprzednim wątku? żałosne..., to w taki sposób wygrywacie w dyskusji? poprzez trolowanie a potem blokowanie przeciwnika w dyskusji, żeby nie mógł dalej celnie ripostować i się bronić...zupełnie jak Łukaszenka na Białorusi...
- może zamiast siedzieć tutaj na githubie to zatrudnijcie się gdzieś jako "ruskie" trolle, albo jedźcie na Białoruś pomóc Łukaszence w trollowani ludzi, że wybory nie były sfałszowane, tak czy inaczej czas żeby w końcu wezwać po was ambulans...
- porobiłam zrzuty ekranu i zapisałam dyskusje, jeszcze jeden głupi komentarz albo manipulacja w moją stronę lub zablokowanie mi znowu wątku, to zgłaszam was do github za nękanie, piszę poważnie, a github traktuje nękanie poważnie...żebyście się nie zdziwili gdy to wam konta polecą razem ze wszystkimi zgłoszeniami i komentarzami, serio chcecie ryzykować i stracić wieloletnie konta...
- @ admin czy możesz zareagować...bo moje prośby do nich nie docierają...ja tu tylko dużo pomagam, zajmuję się zgłoszeniami, w ogóle ich nie zaczepiam i się z nimi generalnie nie zadaję i ich unikam, od dłuższego czasu się normalnie zachowuję, a pomimo tego dokuczają mi...
- tu też xxcriticxx odstawia ciekawe numery: https://github.com/MajkiIT/polish-ads-filter/issues/17775

**Comment by xxcriticxx** [comment, marked as answer]
@zaba-monika nie dropsy, konczymy ta impreze
